test/python/circuit/test_circuit_operations.py

```diff
@@ -799,6 +799,19 @@ def test_reverse_bits_with_registers(self):
 
         self.assertEqual(qc.reverse_bits(), expected)
 
+    def test_reverse_bits_isolated_qubits(self):
+        """Test reversing order of bits when QuantumCircuit is made with isolated Qubits (not in a register)."""
+        from qiskit.circuit import Qubit
+        qubit1 = Qubit()
+        qubit2 = Qubit()
+        qc = QuantumCircuit([qubit1, qubit2])
+        qc.h(0)
+
+        expected = QuantumCircuit([qubit2, qubit1])
+        expected.h(1)
+
+        self.assertEqual(qc.reverse_bits(), expected)
```

Comment:
We should add a couple of tests which also include quantum registers, classical registers and loose classical bits as well.

Reply:
I agree with @jakelishman . We should have a few more tests. I am just thinking out loud. We could have some test that cover circuits initialized with registerless qubits in between different quantum registers just to ensure that the ordering of the registerless qubits is not messed up on using `.reverse_bits()`. I would also add registerless clbits to circuit and use classical conditioned gates that use those registerless clbits. And also probably a test with circuit that only contains registerless qubits and registerless clbits.

Reply:
Thanks for the comments. I've added a new test that includes quantum registers, loose qubits, and classical registers to make sure the functionality is correct. I ran a test with "loose" clbits as well but it errored out when trying to print the circuit to the console (it made the circuit just fine, the error was in the circuit draw function i'm pretty sure). 

I feel like fixing the clbits printing should be its own issue though, what do you all think? @jakelishman @TharrmashasthaPV ?